src/App.jsx

```diff
@@ -36,7 +36,7 @@ const App = () => (
         <Route exact path="/test">
           <ViolenceTestPage />
         </Route>
-        <Route exact path="/jobs">
+        <Route exact path="/opportunities">
           <JobsTrainingPage />
```

Comment:
can you rename this page to OpportunitiesPage please
and anywhere JobsTraining is used just call it Opportunities; the latter is a much better name.

in a separate PR